nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/resources/docs/org.apache.nifi.processors.standard.RecordStats/additionalDetails.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<head>
+    <meta charset="utf-8" />
+    <title>RecordStats</title>
+
+    <link rel="stylesheet" href="../../../../../css/component-usage.css" type="text/css" />
+</head>
+<body>
+    <p>This processor takes in a record set and counts both the overall count and counts that are defined as dynamic properties
+    that map a property name to a record path. Record path counts are provided at two levels:</p>
+    <ul>
+        <li>The overall count of all records that successfully evaluated a record path.</li>
+        <li>A breakdown of counts of unique values that matched the record path operation.</li>
+    </ul>
+    <p>Consider the following record structure:</p>
+    <pre>
+        {
+            "sport": "Soccer",
+            "name": "John Smith"
+        }
+    </pre>
+    <p>A valid mapping here would be <em>sport => /sport</em>.</p>
+    <p>For a record set with JSON like that, five entries and 3 instances of soccer and two instances of football, it would set the following
+    attributes:</p>
+    <ul>
+        <li>record_count: 5</li>
+        <li>sport: 5</li>
+        <li>sport.Soccer: 3</li>
+        <li>sport.Football: 2</li>
```

Comment:
These property names should be more self-descriptive and not to overlap other property name spaces.
I suggest following names:

|current|suggestion|
|-------|------------|
|record_count|recordStats.count|
|sport|recordStats.sport.count|
|sport.Soccer|recordStats.sport.count.Soccer|
|sport.Football|recordStats.sport.count.Football|

Then we can add more stats later, such as recordStats.age.min or recordStats.age.max ... etc

Reply:
Done.